src/test/java/teammates/test/cases/testdriver/TestNgTest.java

```diff
@@ -57,7 +58,7 @@ public void checkTestsInTestNg() throws IOException {
      * @param filesExcludedFromTestNg    Files to be excluded
      * @return                           Files to be checked after excluding tests
      */
-    private HashMap<String, String> excludeFilesNotInTestNg(HashMap<String, String> testFiles,
+    private Map<String, String> excludeFilesNotInTestNg(Map<String, String> testFiles,
                                                             String... filesExcludedFromTestNg) {
```

Comment:
Can fix the indentations here; use 8 spaces instead of aligning as done before. Alternatively see if it fits the previous line.